### PR TITLE
SPCTR-1226 Fix: Select input field text overlapping issue

### PR DESCRIPTION
**Pull request by sushmak02**
### Description
Select Control: Fields Do Not Display Some Words Properly

“g“ in the word large seems to be not displayed fully in the field if possible.

### Screenshots
Issue - https://share.getcloudapp.com/RBumyXK4
Fixed - https://share.getcloudapp.com/llu064xP

### Types of changes

Bug fix (non-breaking change which fixes an issue)

### How has this been tested?

- Add post timeline block
- Go to general → Image
- Select Image size to large

### Checklist:
- [x] My code is tested
- [ ] My code follows accessibility standards 
- [ ] My code has proper inline documentation 
- [ ] I've included any necessary tests 
- [ ] I've included developer documentation 
- [ ] I've added proper labels to this pull request

### CODE REVIEW

File: src/common-editor.scss [hunk 1]
```diff
@@ -447,8 +447,13 @@ label.wp-block-uagb-blockquote__via-label svg {
 	vertical-align: middle;
 }
 
-.components-select-control__input {
+// Select control text going out of screen.
+
+.uagb-inspector-tab .components-select-control .components-input-control__container .components-select-control__input {
 	text-transform: capitalize;
+	line-height: unset;
+	height: 35px;
```
Comment (bsf-zanev) [inline src/common-editor.scss:455]: As per the Spectra Style Guide, the inherited height of all select controls should be exactly 30px.

Remove all of the newly added CSS properties, and in place of that use "line-height: 30px".

Test the same across Spectra.

**Review by bsf-zanev — CHANGES_REQUESTED** [review]
Do resolve the issues noted in the pending comment.

File: src/common-editor.scss (hunk 1 above)
Comment (sushmak02) [reply src/common-editor.scss]: I have updated the CSS @bsf-zanev . Please review the PR

**Review by bsf-zanev — APPROVED** [review]
Updates Adhere to Style Guide.

**Pull request merged**